urllib3/util/__init__.py

```diff
@@ -6,6 +6,7 @@
 from .ssl_ import (
     SSLContext,
     HAS_SNI,
+    IS_PYOPENSSL,
```

Comment:
Yeah, I thought about that, but they're not really the same kind of thing. `HAS_SNI` is feature detection: `IS_PYOPENSSL` is mostly just a warning flag (and in this case only really exists for testing at this time, though we may want to use it later).